lib/sdp/importers/cdc.rb

```diff
@@ -0,0 +1,17 @@
+module SDP
+  module Importers
+    module CDC
+      def self.import_programs(program_csv_file)
+        CSV.foreach(program_csv_file, headers: :first_row) do |row|
+          SurveillanceProgram.create!(row.to_hash)
```

Comment:
I think these be find_or_create_by instead of straight up create that way it shouldn't error out if the csv file has entries in it that are already in the db